runtime/services/lightd/service.js

```diff
@@ -575,6 +575,10 @@ Light.prototype.setPickup = function (appId, duration, withAwaken) {
   })
 }
 
+Light.prototype.setGlobalAlphaFactor = function (alphaFactor) {
+  manager.setGlobalAlphaFactor(alphaFactor)
+}
```

Comment:
As I mentioned at service, I think this is unnecessary to proxy these methods from manager.